themes/aframe/source/css/_settings.styl

```diff
@@ -17,6 +17,7 @@ $code-font-size = 0.8rem
 
 // Colors
 $dark   = #2c3e50
+$fair = lighten(#2c3e50, 20%)
```

Comment:
was trying to differentiate here too:

![image](https://cloud.githubusercontent.com/assets/203725/15687676/d43ae842-272a-11e6-87b1-fa29f0c87466.png)

![image](https://cloud.githubusercontent.com/assets/203725/15687720/0d38ddf2-272b-11e6-9812-61dbdd0adf0a.png)